### PR TITLE
NetHui Access Pass SignIn

### PR DESCRIPTION
**Pull request by rafaelbernard**
- Add sign in with IRMA
- Add signin Online or In Person

### CODE REVIEW

File: election.html [hunk 1]
```diff
@@ -268,15 +270,23 @@ <h3>Select your favorite cake!</h1>
             <input type="button" class="btn btn-info" v-on:click="reset" value="Reset">
         </div>
 
-        <footer class="my-5 pt-5 text-muted text-center text-small">
+        <div class="p-5 text-center">
+            <ul class="list-inline">
+                <li class="list-inline-item"><a href="simple-issuer.html">Issue Address and Email</a></li>
```
Comment (ehsan-fj) [inline election.html]: `Full name` and email?

File: event-access-pass.html [hunk 2]
```diff
@@ -318,106 +372,139 @@ <h2 class="page-header">NetHui 2020</h2>
...
     </div>
 
     <div class="text-right">
-        <input type="button" class="btn btn-info" v-on:click="reset" value="Reset">
+        <input type="button" class="btn btn-info" v-on:click="reset()" value="Reset">
     </div>
 
     <div class="p-5 text-center">
         <ul class="list-inline">
+            <li class="list-inline-item"><a href="simple-issuer.html">Issue Address and Email</a></li>
```
Comment (ehsan-fj) [inline event-access-pass.html]: `Full name`?

**Review by ehsan-fj — APPROVED** [review]

File: election.html (hunk 1 above)
Comment (rafaelbernard) [reply election.html]: Good catch!

**Comment by ehsan-fj** [comment]
yup that's good to have address issuer there 👍

**Pull request merged**